rpc/wrtc_client.go

```diff
@@ -433,3 +438,12 @@ func dialSignalingServer(
 	conn, _, err := dialDirectGRPC(ctx, signalingServer, dOpts, logger)
 	return conn, err
 }
+
+func filterEOF(err error, logger utils.ZapCompatibleLogger) error {
+	s, isGRPCErr := status.FromError(err)
+	if errors.Is(err, io.EOF) || (isGRPCErr && (s.Code() == codes.Internal && strings.Contains(s.Message(), "EOF"))) {
```

Comment:
was not sure if this should be `io.EOF` errors since that is also what I see in the flaky test logs or grpc EOF errors

Reply:
What you have looks good to me; I'm not sure gRPC EOF's actually wrap `io.EOF`, so your code here is probably necessary.